libs/ledger-live-common/src/wallet-api/Exchange/tracking.ts

```diff
@@ -30,47 +29,52 @@ function getEventData(manifest: AppManifest) {
 // in order to get the exact type matching the tracking wrapper API
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export default function trackingWrapper(trackCall: TrackExchange) {
-  const track = (event: string, properties: Record<string, any> | null) =>
-    trackCall(event, properties, null);
+  const track = (event: string, properties: Record<string, string> | null) => {
+    return trackCall(event, properties, null);
+  };
 
   return {
     // Generate Exchange nonce modal open
-    startExchangeRequested: (manifest: AppManifest) => {
-      track("WalletAPI start Exchange Nonce request", getEventData(manifest));
+    startExchangeRequested: ({ provider, exchangeType }: TrackEventPayload) => {
+      track(`Starts Exchange ${exchangeType} Nonce request`, { provider, exchangeType });
     },
 
     // Successfully generated an Exchange app nonce
-    startExchangeSuccess: (manifest: AppManifest) => {
-      track("WalletAPI start Exchange Nonce success", getEventData(manifest));
+    startExchangeSuccess: ({ provider, exchangeType }: TrackEventPayload) => {
+      track(`Starts Exchange ${exchangeType} Nonce success`, { provider, exchangeType });
     },
 
     // Failed to generate an Exchange app nonce
-    startExchangeFail: (manifest: AppManifest) => {
-      track("WalletAPI start Exchange Nonce fail", getEventData(manifest));
+    startExchangeFail: ({ provider, exchangeType }: TrackEventPayload) => {
+      track(`Starts Exchange ${exchangeType} Nonce fail`, { provider, exchangeType });
     },
 
     // No Params to generate an Exchange app nonce
     startExchangeNoParams: (manifest: AppManifest) => {
-      track("WalletAPI start Exchange no params", getEventData(manifest));
+      track("Start Exchange no params", getEventData(manifest));
```

Comment:
Should it be `Starts` like the other ?
I feel like `Start` would be more correct for all